### PR TITLE
Added bubble_sort

### PR DESCRIPTION
**Pull request by navneet15**
Fixes #{ISSUE_NUMBER}

#### By submitting this pull request I confirm I've read and complied with the below declarations.

- [x] I have read the [Contribution guidelines](https://github.com/iiitv/algos/blob/master/CONTRIBUTING.md) and I am confident that my PR reflects them.
- [x] I have followed the [coding guidelines](https://github.com/iiitv/algos/blob/master/CONTRIBUTING.md#cs) for this project.
- [x] My code follows the [skeleton code structure](https://github.com/iiitv/algos/blob/master/CONTRIBUTING.md#sample).
- [x] This pull request has a descriptive title. For example, `Added {Algorithm/DS name} [{Language}]`, not `Update README.md` or `Added new code`.
- [x] This pull request will be closed if I fail to update it even once in a continuous time span of 7 days.
- [x] This pull request shall only be reviewed and merged once the Travis build passes. No maintainer or supporter shall be obliged to review it before this condition is met.
- [x] I have mentioned the issue number correctly (with hyperlink) in this pull request description.

### CODE REVIEW

**Comment by singhpratyush** [comment]
Hi! Thanks for showing interest in contributing to this repository.

Please make sure that you have ticked the points mentioned in PR description correctly.

Thanks again!

File: bubble_sort/bubble_sort.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# Python program for implementation of Bubble Sort
+
```
Comment (singhpratyush) [inline bubble_sort/bubble_sort.py]: Line contains following spacing inconsistencies:
- Trailing whitespaces.

*Origin: SpaceConsistencyBear, Section: `python spacing`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmpwl1l6gsj/bubble_sort/bubble_sort.py
+++ b/tmp/tmpwl1l6gsj/bubble_sort/bubble_sort.py
@@ -1,5 +1,5 @@
 # Python program for implementation of Bubble Sort
- 
+
 def bubbleSort(arr):
     n = len(arr)
  
```

File: bubble_sort/bubble_sort.py [hunk 2]
```diff
@@ -0,0 +1,25 @@
+# Python program for implementation of Bubble Sort
+ 
+def bubbleSort(arr):
+    n = len(arr)
+
```
Comment (singhpratyush) [inline bubble_sort/bubble_sort.py]: Line contains following spacing inconsistencies:
- Trailing whitespaces.

*Origin: SpaceConsistencyBear, Section: `python spacing`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmpwl1l6gsj/bubble_sort/bubble_sort.py
+++ b/tmp/tmpwl1l6gsj/bubble_sort/bubble_sort.py
@@ -2,7 +2,7 @@
  
 def bubbleSort(arr):
     n = len(arr)
- 
+
     # Traverse through all array elements
     for i in range(n):
  
```

File: bubble_sort/bubble_sort.py [hunk 3]
```diff
@@ -0,0 +1,25 @@
+# Python program for implementation of Bubble Sort
+ 
+def bubbleSort(arr):
+    n = len(arr)
+ 
+    # Traverse through all array elements
+    for i in range(n):
+
```
Comment (singhpratyush) [inline bubble_sort/bubble_sort.py]: Line contains following spacing inconsistencies:
- Trailing whitespaces.

*Origin: SpaceConsistencyBear, Section: `python spacing`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmpwl1l6gsj/bubble_sort/bubble_sort.py
+++ b/tmp/tmpwl1l6gsj/bubble_sort/bubble_sort.py
@@ -5,7 +5,7 @@
  
     # Traverse through all array elements
     for i in range(n):
- 
+
         # Last i elements are already in place
         for j in range(0, n-i-1):
  
```

File: bubble_sort/bubble_sort.py [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+ 
+def bubbleSort(arr):
+    n = len(arr)
+ 
+    # Traverse through all array elements
+    for i in range(n):
+ 
+        # Last i elements are already in place
+        for j in range(0, n-i-1):
+
```
Comment (singhpratyush) [inline bubble_sort/bubble_sort.py]: Line contains following spacing inconsistencies:
- Trailing whitespaces.

*Origin: SpaceConsistencyBear, Section: `python spacing`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmpwl1l6gsj/bubble_sort/bubble_sort.py
+++ b/tmp/tmpwl1l6gsj/bubble_sort/bubble_sort.py
@@ -8,7 +8,7 @@
  
         # Last i elements are already in place
         for j in range(0, n-i-1):
- 
+
             # traverse the array from 0 to n-i-1
             # Swap if the element found is greater
             # than the next element
```

File: bubble_sort/bubble_sort.py [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+ 
+        # Last i elements are already in place
+        for j in range(0, n-i-1):
+ 
+            # traverse the array from 0 to n-i-1
+            # Swap if the element found is greater
+            # than the next element
+            if arr[j] > arr[j+1] :
+                arr[j], arr[j+1] = arr[j+1], arr[j]
+
```
Comment (singhpratyush) [inline bubble_sort/bubble_sort.py]: Line contains following spacing inconsistencies:
- Trailing whitespaces.

*Origin: SpaceConsistencyBear, Section: `python spacing`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmpwl1l6gsj/bubble_sort/bubble_sort.py
+++ b/tmp/tmpwl1l6gsj/bubble_sort/bubble_sort.py
@@ -14,7 +14,7 @@
             # than the next element
             if arr[j] > arr[j+1] :
                 arr[j], arr[j+1] = arr[j+1], arr[j]
- 
+
 # Driver code to test above
 arr = [64, 34, 25, 12, 22, 11, 90]
  
```

File: bubble_sort/bubble_sort.py [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+ 
+            # traverse the array from 0 to n-i-1
+            # Swap if the element found is greater
+            # than the next element
+            if arr[j] > arr[j+1] :
+                arr[j], arr[j+1] = arr[j+1], arr[j]
+ 
+# Driver code to test above
+arr = [64, 34, 25, 12, 22, 11, 90]
+
```
Comment (singhpratyush) [inline bubble_sort/bubble_sort.py]: Line contains following spacing inconsistencies:
- Trailing whitespaces.

*Origin: SpaceConsistencyBear, Section: `python spacing`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmpwl1l6gsj/bubble_sort/bubble_sort.py
+++ b/tmp/tmpwl1l6gsj/bubble_sort/bubble_sort.py
@@ -17,7 +17,7 @@
  
 # Driver code to test above
 arr = [64, 34, 25, 12, 22, 11, 90]
- 
+
 bubbleSort(arr)
  
 print ("Sorted array is:")
```

File: bubble_sort/bubble_sort.py [hunk 7]
```diff
@@ -0,0 +1,25 @@
...
+            # Swap if the element found is greater
+            # than the next element
+            if arr[j] > arr[j+1] :
+                arr[j], arr[j+1] = arr[j+1], arr[j]
+ 
+# Driver code to test above
+arr = [64, 34, 25, 12, 22, 11, 90]
+ 
+bubbleSort(arr)
+
```
Comment (singhpratyush) [inline bubble_sort/bubble_sort.py]: Line contains following spacing inconsistencies:
- Trailing whitespaces.

*Origin: SpaceConsistencyBear, Section: `python spacing`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmpwl1l6gsj/bubble_sort/bubble_sort.py
+++ b/tmp/tmpwl1l6gsj/bubble_sort/bubble_sort.py
@@ -19,7 +19,7 @@
 arr = [64, 34, 25, 12, 22, 11, 90]
  
 bubbleSort(arr)
- 
+
 print ("Sorted array is:")
 for i in range(len(arr)):
     print ("%d" %arr[i]), 
```

File: bubble_sort/bubble_sort.py [hunk 8]
```diff
@@ -0,0 +1,25 @@
...
+                arr[j], arr[j+1] = arr[j+1], arr[j]
+ 
+# Driver code to test above
+arr = [64, 34, 25, 12, 22, 11, 90]
+ 
+bubbleSort(arr)
+ 
+print ("Sorted array is:")
+for i in range(len(arr)):
+    print ("%d" %arr[i]),
```
Comment (singhpratyush) [inline bubble_sort/bubble_sort.py]: Line contains following spacing inconsistencies:
- Trailing whitespaces.

*Origin: SpaceConsistencyBear, Section: `python spacing`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmpwl1l6gsj/bubble_sort/bubble_sort.py
+++ b/tmp/tmpwl1l6gsj/bubble_sort/bubble_sort.py
@@ -22,4 +22,4 @@
  
 print ("Sorted array is:")
 for i in range(len(arr)):
-    print ("%d" %arr[i]), 
+    print ("%d" %arr[i]),
```

File: bubble_sort/bubble_sort.py (hunk 1 above)
Comment (singhpratyush) [inline bubble_sort/bubble_sort.py]: The code does not comply to PEP8.

*Origin: PEP8Bear, Section: `pep8`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmpwl1l6gsj/bubble_sort/bubble_sort.py
+++ b/tmp/tmpwl1l6gsj/bubble_sort/bubble_sort.py
@@ -1,5 +1,6 @@
 # Python program for implementation of Bubble Sort
- 
+
+
 def bubbleSort(arr):
     n = len(arr)
  
```

File: bubble_sort/bubble_sort.py (hunk 2 above)
Comment (singhpratyush) [inline bubble_sort/bubble_sort.py]: The code does not comply to PEP8.

*Origin: PEP8Bear, Section: `pep8`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmpwl1l6gsj/bubble_sort/bubble_sort.py
+++ b/tmp/tmpwl1l6gsj/bubble_sort/bubble_sort.py
@@ -2,7 +2,7 @@
  
 def bubbleSort(arr):
     n = len(arr)
- 
+
     # Traverse through all array elements
     for i in range(n):
  
```

File: bubble_sort/bubble_sort.py (hunk 3 above)
Comment (singhpratyush) [inline bubble_sort/bubble_sort.py]: The code does not comply to PEP8.

*Origin: PEP8Bear, Section: `pep8`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmpwl1l6gsj/bubble_sort/bubble_sort.py
+++ b/tmp/tmpwl1l6gsj/bubble_sort/bubble_sort.py
@@ -5,7 +5,7 @@
  
     # Traverse through all array elements
     for i in range(n):
- 
+
         # Last i elements are already in place
         for j in range(0, n-i-1):
  
```

File: bubble_sort/bubble_sort.py (hunk 4 above)
Comment (singhpratyush) [inline bubble_sort/bubble_sort.py]: The code does not comply to PEP8.

*Origin: PEP8Bear, Section: `pep8`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmpwl1l6gsj/bubble_sort/bubble_sort.py
+++ b/tmp/tmpwl1l6gsj/bubble_sort/bubble_sort.py
@@ -8,7 +8,7 @@
  
         # Last i elements are already in place
         for j in range(0, n-i-1):
- 
+
             # traverse the array from 0 to n-i-1
             # Swap if the element found is greater
             # than the next element
```

File: bubble_sort/bubble_sort.py [hunk 9]
```diff
@@ -0,0 +1,25 @@
...
+    # Traverse through all array elements
+    for i in range(n):
+ 
+        # Last i elements are already in place
+        for j in range(0, n-i-1):
+ 
+            # traverse the array from 0 to n-i-1
+            # Swap if the element found is greater
+            # than the next element
+            if arr[j] > arr[j+1] :
```
Comment (singhpratyush) [inline bubble_sort/bubble_sort.py]: The code does not comply to PEP8.

*Origin: PEP8Bear, Section: `pep8`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmpwl1l6gsj/bubble_sort/bubble_sort.py
+++ b/tmp/tmpwl1l6gsj/bubble_sort/bubble_sort.py
@@ -12,9 +12,10 @@
             # traverse the array from 0 to n-i-1
             # Swap if the element found is greater
             # than the next element
-            if arr[j] > arr[j+1] :
+            if arr[j] > arr[j+1]:
                 arr[j], arr[j+1] = arr[j+1], arr[j]
- 
+
+
 # Driver code to test above
 arr = [64, 34, 25, 12, 22, 11, 90]
  
```

File: bubble_sort/bubble_sort.py (hunk 6 above)
Comment (singhpratyush) [inline bubble_sort/bubble_sort.py]: The code does not comply to PEP8.

*Origin: PEP8Bear, Section: `pep8`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmpwl1l6gsj/bubble_sort/bubble_sort.py
+++ b/tmp/tmpwl1l6gsj/bubble_sort/bubble_sort.py
@@ -17,9 +17,9 @@
  
 # Driver code to test above
 arr = [64, 34, 25, 12, 22, 11, 90]
- 
+
 bubbleSort(arr)
- 
-print ("Sorted array is:")
+
+print("Sorted array is:")
 for i in range(len(arr)):
-    print ("%d" %arr[i]), 
+    print("%d" % arr[i]),
```

**Comment by singhpratyush** [comment]
Comment on 35905a665d782702a5b99206b667ae7bd8aa69c0, file bubble_sort/bubble_sort.py, line 2.

Line contains following spacing inconsistencies:
- Trailing whitespaces.

*Origin: SpaceConsistencyBear, Section: `python spacing`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmpnrqvgtsu/bubble_sort/bubble_sort.py
+++ b/tmp/tmpnrqvgtsu/bubble_sort/bubble_sort.py
@@ -1,5 +1,5 @@
 # Python program for implementation of Bubble Sort
- 
+
 def bubbleSort(arr):
     n = len(arr)
  
```

**Comment by singhpratyush** [comment]
Comment on 35905a665d782702a5b99206b667ae7bd8aa69c0, file bubble_sort/bubble_sort.py, line 5.

Line contains following spacing inconsistencies:
- Trailing whitespaces.

*Origin: SpaceConsistencyBear, Section: `python spacing`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmpnrqvgtsu/bubble_sort/bubble_sort.py
+++ b/tmp/tmpnrqvgtsu/bubble_sort/bubble_sort.py
@@ -2,7 +2,7 @@
  
 def bubbleSort(arr):
     n = len(arr)
- 
+
     # Traverse through all array elements
     for i in range(n):
  
```

**Comment by singhpratyush** [comment]
Comment on 35905a665d782702a5b99206b667ae7bd8aa69c0, file bubble_sort/bubble_sort.py, line 8.

Line contains following spacing inconsistencies:
- Trailing whitespaces.

*Origin: SpaceConsistencyBear, Section: `python spacing`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmpnrqvgtsu/bubble_sort/bubble_sort.py
+++ b/tmp/tmpnrqvgtsu/bubble_sort/bubble_sort.py
@@ -5,7 +5,7 @@
  
     # Traverse through all array elements
     for i in range(n):
- 
+
         # Last i elements are already in place
         for j in range(0, n-i-1):
  
```

**Comment by singhpratyush** [comment]
Comment on 35905a665d782702a5b99206b667ae7bd8aa69c0, file bubble_sort/bubble_sort.py, line 11.

Line contains following spacing inconsistencies:
- Trailing whitespaces.

*Origin: SpaceConsistencyBear, Section: `python spacing`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmpnrqvgtsu/bubble_sort/bubble_sort.py
+++ b/tmp/tmpnrqvgtsu/bubble_sort/bubble_sort.py
@@ -8,7 +8,7 @@
  
         # Last i elements are already in place
         for j in range(0, n-i-1):
- 
+
             # traverse the array from 0 to n-i-1
             # Swap if the element found is greater
             # than the next element
```

**Comment by singhpratyush** [comment]
Comment on 35905a665d782702a5b99206b667ae7bd8aa69c0, file bubble_sort/bubble_sort.py, line 17.

Line contains following spacing inconsistencies:
- Trailing whitespaces.

*Origin: SpaceConsistencyBear, Section: `python spacing`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmpnrqvgtsu/bubble_sort/bubble_sort.py
+++ b/tmp/tmpnrqvgtsu/bubble_sort/bubble_sort.py
@@ -14,7 +14,7 @@
             # than the next element
             if arr[j] > arr[j+1] :
                 arr[j], arr[j+1] = arr[j+1], arr[j]
- 
+
 # Driver code to test above
 arr = [64, 34, 25, 12, 22, 11, 90]
  
```

**Comment by singhpratyush** [comment]
Comment on 35905a665d782702a5b99206b667ae7bd8aa69c0, file bubble_sort/bubble_sort.py, line 20.

Line contains following spacing inconsistencies:
- Trailing whitespaces.

*Origin: SpaceConsistencyBear, Section: `python spacing`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmpnrqvgtsu/bubble_sort/bubble_sort.py
+++ b/tmp/tmpnrqvgtsu/bubble_sort/bubble_sort.py
@@ -17,7 +17,7 @@
  
 # Driver code to test above
 arr = [64, 34, 25, 12, 22, 11, 90]
- 
+
 bubbleSort(arr)
  
 print ("Sorted array is:")
```

**Comment by singhpratyush** [comment]
Comment on 35905a665d782702a5b99206b667ae7bd8aa69c0, file bubble_sort/bubble_sort.py, line 22.

Line contains following spacing inconsistencies:
- Trailing whitespaces.

*Origin: SpaceConsistencyBear, Section: `python spacing`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmpnrqvgtsu/bubble_sort/bubble_sort.py
+++ b/tmp/tmpnrqvgtsu/bubble_sort/bubble_sort.py
@@ -19,7 +19,7 @@
 arr = [64, 34, 25, 12, 22, 11, 90]
  
 bubbleSort(arr)
- 
+
 print ("Sorted array is:")
 for i in range(len(arr)):
     print ("%d" %arr[i]), 
```

**Comment by singhpratyush** [comment]
Comment on 35905a665d782702a5b99206b667ae7bd8aa69c0, file bubble_sort/bubble_sort.py, line 25.

Line contains following spacing inconsistencies:
- Trailing whitespaces.

*Origin: SpaceConsistencyBear, Section: `python spacing`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmpnrqvgtsu/bubble_sort/bubble_sort.py
+++ b/tmp/tmpnrqvgtsu/bubble_sort/bubble_sort.py
@@ -22,4 +22,4 @@
  
 print ("Sorted array is:")
 for i in range(len(arr)):
-    print ("%d" %arr[i]), 
+    print ("%d" %arr[i]),
```

**Comment by singhpratyush** [comment]
Comment on 35905a665d782702a5b99206b667ae7bd8aa69c0, file bubble_sort/bubble_sort.py, line 2.

The code does not comply to PEP8.

*Origin: PEP8Bear, Section: `pep8`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmpnrqvgtsu/bubble_sort/bubble_sort.py
+++ b/tmp/tmpnrqvgtsu/bubble_sort/bubble_sort.py
@@ -1,5 +1,6 @@
 # Python program for implementation of Bubble Sort
- 
+
+
 def bubbleSort(arr):
     n = len(arr)
  
```

**Comment by singhpratyush** [comment]
Comment on 35905a665d782702a5b99206b667ae7bd8aa69c0, file bubble_sort/bubble_sort.py, line 5.

The code does not comply to PEP8.

*Origin: PEP8Bear, Section: `pep8`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmpnrqvgtsu/bubble_sort/bubble_sort.py
+++ b/tmp/tmpnrqvgtsu/bubble_sort/bubble_sort.py
@@ -2,7 +2,7 @@
  
 def bubbleSort(arr):
     n = len(arr)
- 
+
     # Traverse through all array elements
     for i in range(n):
  
```

**Comment by singhpratyush** [comment]
Comment on 35905a665d782702a5b99206b667ae7bd8aa69c0, file bubble_sort/bubble_sort.py, line 8.

The code does not comply to PEP8.

*Origin: PEP8Bear, Section: `pep8`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmpnrqvgtsu/bubble_sort/bubble_sort.py
+++ b/tmp/tmpnrqvgtsu/bubble_sort/bubble_sort.py
@@ -5,7 +5,7 @@
  
     # Traverse through all array elements
     for i in range(n):
- 
+
         # Last i elements are already in place
         for j in range(0, n-i-1):
  
```

**Comment by singhpratyush** [comment]
Comment on 35905a665d782702a5b99206b667ae7bd8aa69c0, file bubble_sort/bubble_sort.py, line 11.

The code does not comply to PEP8.

*Origin: PEP8Bear, Section: `pep8`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmpnrqvgtsu/bubble_sort/bubble_sort.py
+++ b/tmp/tmpnrqvgtsu/bubble_sort/bubble_sort.py
@@ -8,7 +8,7 @@
  
         # Last i elements are already in place
         for j in range(0, n-i-1):
- 
+
             # traverse the array from 0 to n-i-1
             # Swap if the element found is greater
             # than the next element
```

**Comment by singhpratyush** [comment]
Comment on 35905a665d782702a5b99206b667ae7bd8aa69c0, file bubble_sort/bubble_sort.py, line 15.

The code does not comply to PEP8.

*Origin: PEP8Bear, Section: `pep8`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmpnrqvgtsu/bubble_sort/bubble_sort.py
+++ b/tmp/tmpnrqvgtsu/bubble_sort/bubble_sort.py
@@ -12,9 +12,10 @@
             # traverse the array from 0 to n-i-1
             # Swap if the element found is greater
             # than the next element
-            if arr[j] > arr[j+1] :
+            if arr[j] > arr[j+1]:
                 arr[j], arr[j+1] = arr[j+1], arr[j]
- 
+
+
 # Driver code to test above
 arr = [64, 34, 25, 12, 22, 11, 90]
  
```

**Comment by singhpratyush** [comment]
Comment on 35905a665d782702a5b99206b667ae7bd8aa69c0, file bubble_sort/bubble_sort.py, line 20.

The code does not comply to PEP8.

*Origin: PEP8Bear, Section: `pep8`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmpnrqvgtsu/bubble_sort/bubble_sort.py
+++ b/tmp/tmpnrqvgtsu/bubble_sort/bubble_sort.py
@@ -17,9 +17,9 @@
  
 # Driver code to test above
 arr = [64, 34, 25, 12, 22, 11, 90]
- 
+
 bubbleSort(arr)
- 
-print ("Sorted array is:")
+
+print("Sorted array is:")
 for i in range(len(arr)):
-    print ("%d" %arr[i]), 
+    print("%d" % arr[i]),
```

**Comment by mohitkyadav** [comment]
@navneet15 Please open an issue before sending a PR, and read the [contribution guidelines](https://github.com/iiitv/algos/blob/master/CONTRIBUTING.md). One issue for one algorithm.

**Pull request closed without merging**